src/web/client/extension.ts

```diff
@@ -265,6 +265,7 @@ export function processWorkspaceStateChanges(context: vscode.ExtensionContext) {
                                     entityInfo
                                 });
                             }
+                            WebExtensionContext.quickPickProvider.updateQuickPickItems(entityInfo);
```

Comment:
please move this entire logic under "if (isCoPresenceEnabled()) {" to its own function. This funciton is getting way to large .